cms/templates/export.html

```diff
@@ -1,5 +1,5 @@
-<%inherit file="base.html" />
-<%def name="online_help_token()"><% return "export" %></%def>
+<%def name="online_help_token1()"><% return "export_library" %></%def>
+<%def name="online_help_token2()"><% return "export_course" %></%def>
```

Comment:
@catong I would use a more explicit name than numbering the tokens, that would make it more obvious which one we are using below in each seaction - maybe `online_help_token_library` and `online_help_token_course`?